src/V3SchedReplicate.cpp

```diff
@@ -46,24 +46,28 @@ VL_DEFINE_DEBUG_FUNCTIONS;
 
 namespace V3Sched {
 
-namespace {
-
 // Driving region flags
+// Keeping this in a namespace to avoid name collision of `INPUT` with a symbol from `windows.h`.
 enum RegionFlags : uint8_t {
     NONE = 0x0,  //
     INPUT = 0x1,  // Variable/logic is driven from top level input
     ACTIVE = 0x2,  // Variable/logic is driven from 'act' region logic
     NBA = 0x4  // Variable/logic is driven from 'nba' region logic
 };
 
+}  // namespace V3Sched
+
+using V3Sched::RegionFlags;
+
 //##############################################################################
 // Data structures (graph types)
 
-class Vertex VL_NOT_FINAL : public V3GraphVertex {
-    RegionFlags m_drivingRegions{NONE};  // The regions driving this vertex
+class SchedReplicateVertex VL_NOT_FINAL : public V3GraphVertex {
+    VL_RTTI_IMPL(SchedReplicateVertex, V3GraphVertex)
+    RegionFlags m_drivingRegions{RegionFlags::NONE};  // The regions driving this vertex
```

Comment:
I moved them in the original implementation because I needed to predeclare them in `V3Graph`, and also it was easier than figuring out the namespace in `astgen`. But with this new solution it seems like they could be moved back.

Reply:
I would prefer them the way they were then, so this patch does not introduce unnecessary changes. @wsnyder ?

Reply:
For sure, but let's hear from @mglb if it is really unnecessary, as he took over development on this.

Reply:
This change is no longer needed. I've restored namespaces as they were before.